### PR TITLE
ok60 split rshift layout fix

### PR DESCRIPTION
**Pull request by orhid**
Fixes a mislabelled key in the split rshift keymap.

## Description

Names of layouts are changed to match community layout names. A single key was mislabelled in a layout. Also adding my own keymap.

## Types of Changes

- [ ] Core
- [ x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ok60/keymaps/orhid/keymap.c
Comment (drashna) [inline keyboards/ok60/keymaps/orhid/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by orhid** [comment]
Is this one good enough?

**Comment by orhid** [comment]
any chance this will get merged?

**Comment by drashna** [comment]
> any chance this will get merged?

Once the merge conflicts are fixed, then, it still needs a second review.

**Comment by orhid** [comment]
oh i see, somebody managed to change ok60 before this got approved. i'll see maybe they fixed this bug also.

**Comment by orhid** [comment]
the bug actually got fixed, so I am withdrawing this PR

**Pull request closed without merging**
